checklists/alz_checklist.en.json

```diff
@@ -1220,127 +1183,309 @@
+            "id": "D07.16",
+            "severity": "Low",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/premium-features#web-categories"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Use Azure Firewall in parallel/sequence with Azure App Gateway",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "6eff7e6c-6c4a-43d7-be3f-6641c2cb3d4a",
+            "id": "D07.17",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/architecture/example-scenario/gateway/firewall-application-gateway#firewall-and-application-gateway-in-parallel"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Enable Azure Firewall DNS proxy configuration ",
+            "waf": "Security",
+            "service": "Firewall",
+            "guid": "94f3eede-9aa3-4088-92a3-bb9a56509fad",
+            "id": "D07.18",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/dns-details"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Restrict usage of Public IP addresses directly tied to Virtual Machines",
+            "waf": "Security",
+            "service": "Firewall",
+            "guid": "3c5a808d-c695-4c14-a63c-c7ab7a510e41",
+            "id": "D07.19",
+            "severity": "Medium",
+            "link": "https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Policies#corp"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Integrate Azure Firewall with Azure Monitor",
+            "waf": "Monitor",
+            "service": "Firewall",
+            "guid": "1dc04554-dece-4ffb-a49e-5c683e09f8da",
+            "id": "D07.20",
+            "severity": "Low",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/firewall-diagnostics"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Use Microsoft Defender for Cloud",
+            "waf": "Monitor",
+            "service": "Firewall",
+            "guid": "6e9ad7f6-445a-4286-949f-dbbeb7727c1e",
+            "id": "D07.21",
+            "severity": "Low",
+            "link": "https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-workspace-health#view-log-analytics-workspace-health-and-set-up-health-status-alerts"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Automate Backups",
+            "waf": "Monitor",
```

Comment:
We can make this task something like "Implement backups for your firewall rules" instead of just automate backups.